Source/gui/SNTBinaryMessageWindowView.swift

```diff
@@ -85,6 +85,19 @@ func copyDetailsToClipboard(e: SNTStoredEvent?, customURL: String?) {
   pasteboard.setString(s, forType: .string)
 }
 
+public func CopyDetailsButton(e: SNTStoredEvent?, customURL: String?) -> some View {
+  Button(action: { copyDetailsToClipboard(e: e, customURL: customURL as String?) }) {
+    HStack(spacing: 2.0) {
+      Text("Copy Details", comment: "Copy Details button in more details dialog")
```

Comment:
The comment is technically not correct anymore

Reply:
Updated